files.js/modal.js

```diff
@@ -0,0 +1,165 @@
+    projectButton: 'See Project',
+  },
+  {
+    id: 'proj1',
+    projectHeader: 'Profesional Art Printing Data',
+    projectDesc: 'A daily selection of privately personalized reads; no accounts orsign-ups required. has been the industrys standard',
+    projectHtml: 'html',
+    projectbootstrap: 'bootstrap',
+    projectRuby: 'ruby',
+    projectButton: 'See Project',
+  },
+  {
+    id: 'proj1',
+    projectHeader: 'Profesional Art Printing Data',
+    projectDesc: 'A daily selection of privately personalized reads; no accounts orsign-ups required. has been the industrys standard',
+    projectHtml: 'html',
+    projectbootstrap: 'bootstrap',
+    projectRuby: 'ruby',
+    projectButton: 'See Project',
+  },
+  {
+    id: 'proj1',
+    projectHeader: 'Profesional Art Printing Data',
+    projectDesc: 'A daily selection of privately personalized reads; no accounts orsign-ups required. has been the industrys standard',
+    projectHtml: 'html',
+    projectbootstrap: 'bootstrap',
+    projectRuby: 'ruby',
+    projectButton: 'See Project',
+  },
+  {
+    id: 'proj1',
+    projectHeader: 'Profesional Art Printing Data',
+    projectDesc: 'A daily selection of privately personalized reads; no accounts orsign-ups required. has been the industrys standard',
+    projectHtml: 'html',
+    projectbootstrap: 'bootstrap',
+    projectRuby: 'ruby',
+    projectButton: 'See Project',
+  },
+  {
+    id: 'proj6',
+    title: 'Multi Post Stories',
+    closebtn: '<i id="closeModal" class="fa fa-times" aria-hidden="true"></i>',
+    html: 'html',
+    bootstrap: 'bootstrap',
+    ruby: 'Ruby On Rails',
+    projectImage: 'images/Snapshoot Portfolio.png',
+    projectDesc: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam amet animi voluptate doloribus at quaerat, vitae dicta sunt possimus voluptatum, facilis vel? Eligendi, harum ipsum excepturi sint at quasi? Repellendus eveniet suscipit quibusdam inventore architecto minus eligendi adipisci. Harum atque voluptate dolore fugiat pariatur eius ullam architecto porro aspernatur nihil.',
+    projectbtnlive: 'https://carlylechia.github.io/MY-PORTFOLIO/',
+    projectbtnsource: 'https://github.com/carlylechia/MY-PORTFOLIO.git',
+  },
+];
+
+function modalpopup() {
+  popUp.style.display = 'block';
+  body.appendChild(popUp);
+  const showing = `
+  <div class="popupcont">
+      <div class="popupcontheader">
+        <h2 class="desktophide">${projectList[projectList.length - 1].title}</h2>
+        <h2 class="mobilehide">Keeping track of hundreds  of components website</h2>
+        <button id="closeModal" class="closeModal" draggable = "true">${projectList[projectList.length - 1].closebtn}</button>
+      </div>
+      <div class="popupcontlangs">
+        <ul>
+          <li>${projectList[projectList.length - 1].html}</li>
+          <li>${projectList[projectList.length - 1].bootstrap}</li>
+          <li>${projectList[projectList.length - 1].ruby}</li>
+        </ul>
```

Comment:
- [OPTIONAL]
 I recommend you here to use technologies (HTML, CSS ...) as an array in your `projectList` data, and then you can use `for loop` to iterate over them and keep your logic drier (DRY Principe - Don't Repeat Yourself)